src/test/scala/config/KafkaSettingsSpec.scala

```diff
@@ -3,16 +3,17 @@ package config
 import com.typesafe.config.ConfigFactory
 import org.apache.kafka.clients.CommonClientConfigs
 import org.apache.kafka.clients.admin.AdminClientConfig
-import org.scalatest.{FlatSpec, Matchers}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.flatspec.AnyFlatSpec
 
-class KafkaSettingsSpec extends FlatSpec with Matchers {
+class KafkaSettingsSpec extends AnyFlatSpec with Matchers {
   "KafkaSettingsSpec" should "correctly inject the correct endpoint in AdminClient properties" in {
     val conf = ConfigFactory.load("kafka-without-admin-client.conf")
     val settings = KafkaSettings(conf)
 
     settings.address should ===("localhost:9092")
     settings.commandConfig should ===("test.properties")
-    settings.adminClientProps.get(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG) should ===("localhost:9092")
+    settings.adminClientProps.get(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG).asInstanceOf[String] should ===("localhost:9092")
```

Comment:
Is this cast to String needed? it shouldn't be.

Reply:
IntelliJ was complaining about it so I added it. Tests run without it, so it isn't needed, will remove